src/misc2.c

```diff
@@ -4356,7 +4356,10 @@ build_argv_from_list(list_T *l, char ***argv, int *argc)
 	    int i;
 
 	    for (i = 0; i < *argc; ++i)
+	    {
 		vim_free((*argv)[i]);
+		(*argv)[i] = NULL;
```

Comment:
Can you use VIM_CLEAR() here instead of the above two statements?

Reply:
Ahh, it looks quite better, but this change has already merged. Sorry...

Reply:
@yegappan, @brammool 
Now I found this commit: https://github.com/vim/vim/commit/df1956075d9ff7b5869fb548734bc2a7be2bce44 Thank you very much!